en/guide/serialization.md

```diff
@@ -64,11 +64,11 @@ Byte Index | C version | Content | Value | Explanation
 6 | `uint8_t compid` | Component ID (sender)   | 1 - 255 | ID of *component* sending the message. Used to differentiate *components* in a *system* (e.g. autopilot and a camera). Use appropriate values in [MAV_COMPONENT](../messages/common.md#MAV_COMPONENT). Note that the broadcast address `MAV_COMP_ID_ALL` may not be used in this field as it is an invalid *source* address.
 <span id="v2_msgid"></span>7 to 9 | `uint32_t msgid:24` | Message ID (low, middle, high bytes) | 0 - 16777215 | ID of *message type* in payload. Used to decode data back into message object.
 <span id="v2_payload"></span>10 to (n+10) | `uint8_t payload[max 255]` | [Payload](#payload) | | Message data. Depends on message type (i.e. Message ID) and contents.
-(n+11) to (n+12) | `uint16_t checksum` | [Checksum](#checksum) (low byte, high byte) | | X.25 CRC for message (excluding `magic` byte). Includes [CRC_EXTRA](#crc_extra) byte.
-(n+13) to (n+26) | `uint8_t signature[13]`| [Signature](../guide/message_signing.md) | | (Optional) Signature to ensure the link is tamper-proof.
+(n+10) to (n+11) | `uint16_t checksum` | [Checksum](#checksum) (low byte, high byte) | | X.25 CRC for message (excluding `magic` byte). Includes [CRC_EXTRA](#crc_extra) byte.
```

Comment:
If the previous bit is n+10, why this change?

Reply:
Yes, the previous one was n+10 but this was just the end of the "array". Let's consider the following case: We send a message with no payload (e.g. ACK or something similar). This means the "array" of payload should have a length of 0 so the total size of the packet is 12 (or 25 if used with signature). The last index however is n+12 with n=0 -> 12. In the current description there would be an empty byte as payload and the packet would be too big. (This of course scales with bigger packages but I think this example is the easiest to understand.)
This means the n+10 (the payload tail) is going one byte "further" and is actually already the start of the signature.